src/components/molecules/SfMenuItem/SfMenuItem.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../../css/variables';
+
+$sf-menu-item-icon-spacing: 0.75rem;
+$sf-menu-item-padding: $spacing-big $spacing-extra-big;
+$sf-menu-item-font-size: $font-size-big-mobile;
+$sf-menu-item-font-family: $body-font-family-secondary;
+
+.sf-menu-item {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: $sf-menu-item-padding;
+    font-size: $sf-menu-item-font-size;
+    font-family: $sf-menu-item-font-family;
+    
+    &__group {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    &--left {
```

Comment:
do we have use cases on design for this props? i guess there is always one convention and adjustments like this can be made via css when needed

Reply:
I've removed these modifiers. The only remaining one (apart from default `space-between`) is `center`, example:
![image](https://user-images.githubusercontent.com/28540671/55625786-0c10df80-57aa-11e9-863c-07a218723c6e.png)